Tests/TuistLoaderTests/SwiftPackageManager/PackageInfoMapperTests.swift

```diff
@@ -3866,7 +3866,56 @@ final class PackageInfoMapperTests: TuistUnitTestCase {
         )
     }
 
-    func testMap_whenSwiftPackageHasTestTarget() async throws {
+    func testMap_whenRemoteSwiftPackageHasTestTargets() async throws {
+        // Given
+        let basePath = try temporaryPath()
+        try fileHandler.createFolder(basePath.appending(components: ["Package", "Sources", "Target1"]))
```

Comment:
Let's use [FileSystem](https://github.com/tuist/filesystem) in new tests. `FileHandler` will be removed in the future. Same goes for the other tests.

Reply:
I updated the new tests.